build.sbt

```diff
@@ -135,3 +151,17 @@ lazy val noPublishSettings =
     publish := (),
     publishLocal := ()
   )
+
+def filterScripted(excludeF: PartialFunction[String, Seq[File => PathFinder]]) = Def.taskDyn {
+  val base = sbtTestDirectory.value
+  val allTests = base * AllPassFilter * AllPassFilter filter { _.isDirectory }
+  val exclude = excludeF((sbtBinaryVersion in pluginCrossBuild).value)
+  val tests = exclude.map(_.apply(base)).foldLeft(allTests)(
+    (remaining, excluded) => remaining --- excluded).get
+  def args = tests.map(Path.relativeTo(base)).flatten.mkString(" ")
+
+  if (tests.nonEmpty)
+    Def.task(scripted.toTask(" " + args).value)
```

Comment:
This is a super generic way to filter scripted tests based on @xuwei-k's amazing work on https://github.com/sbt/sbt-site/pull/107. If necessary we can simplify it to filter only the last directory, e.g. under `sbt/sbt-{,web}-scalajs-bundle` and include some defaults such as include tests matching `*sbt-${sbtVersion}*`. Let me know what you think/

Reply:
Based on @julienrf 's comment we can make this filtering based on the content of `project/build.properties`. If the file is missing, run for all sbt versions else parse the binary version and run it if it matches `sbtBinaryVersion in pluginCrossBuild`.